lib/routes/gelbooru/namespace.ts

```diff
@@ -0,0 +1,7 @@
+import type { Namespace } from '@/types';
+
+export const namespace: Namespace = {
+    name: 'Gelbooru | Free Anime and Hentai Gallery',
```

Comment:
```suggestion
    name: 'Gelbooru',
```
`name` is expected to have site name only. Description is not expected.